src/services.js

```diff
@@ -0,0 +1,61 @@
+export function saveTOLocalStorage() {}
+
+export function getData(key = 'tasks') {
+  try {
+    return JSON.parse(localStorage.getItem(key) || '[]');
+  } catch (error) {
+    return localStorage.getItem(key) || [];
+  }
+}
+
+export function createLocalStorage(key = 'tasks', value = []) {
+  if (!getData(key)) {
+    localStorage.setItem(key, JSON.stringify(value));
+  }
+}
+
+export function updateTasks(data) {
+  localStorage.setItem('tasks', JSON.stringify(data));
+}
```

Comment:
- [ ] Kindly add unit test for this function as it is part of this milestone's requirements. 
![image](https://user-images.githubusercontent.com/44057863/179314426-e07aa7b1-271b-4c83-82fe-3835f7377590.png)